File: frontend/src/app/terapija/terapijaLijekGetAllResponse.ts
import {LijekGetAllResponseLijek} from "./lijekGetAllResponse";
import {TerapijaGetAllResponseTerapija} from "./TerapijaGetAllResponse";

export interface TerapijaLijekGetAllResponse{
  terapijeLijekovi:TerapijaLijekGetAllResponseTerapijaLijek[]
}

export interface TerapijaLijekGetAllResponseTerapijaLijek{
   terapijaLijekId:number,
   terapijaId:number,
   lijekId:number,
   lijek:LijekGetAllResponseLijek,
   terapija:TerapijaGetAllResponseTerapija
}

File: DomZaStaraLicaApi/Endpoints/TerapijaLijek/GetAll/TerapijaLijekGetAllResponse.cs
using DomZaStaraLicaApi.Data.Models;

namespace DomZaStaraLicaApi.Endpoints.TerapijaLijek.GetAll
{
    public class TerapijaLijekGetAllResponse
    {
      public List<TerapijaLijekGetAllResponseTerapijaLijek> TerapijeLijekovi { get; set; }
    }

    public class TerapijaLijekGetAllResponseTerapijaLijek
    {
        public int TerapijaLijekId { get; set; }
        public int TerapijaId { get; set; }
        public int? LijekId { get; set; }
        public CLijek? Lijek { get; set; }
        public DomZaStaraLicaApi.Data.Models.Terapija Terapija { get; set; }


    }
}
